src/map/entities/charentity.cpp

```diff
@@ -464,6 +464,40 @@ bool CCharEntity::ReloadParty()
     return m_reloadParty;
 }
 
+void CCharEntity::RemoveTrust(CTrustEntity* PTrust)
+{
+    if (!PTrust->PAI->IsSpawned()) {
+        return;
+    }
```

Comment:
curly brace on new line
although for if blocks with only one statement, we prefer it without curly braces

Reply:
Acknowledged, and will do.

As an aside, I would strongly recommend that curly braces are always enforced, lest you fall foul of the [goto fail](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) bug 0:) though as this is the current standard for this project, I'll omit them.

Reply:
uh, I definitely prefer curly braces for single line ifs for that reason

(on a separate line though)

Reply:
Well make a decision and I'll do whatever. Currently it has no braces.